src/3d/qgs3daxissettings.cpp

```diff
@@ -80,16 +92,26 @@ void Qgs3DAxisSettings::readXml( const QDomElement &element, const QgsReadWriteC
 void Qgs3DAxisSettings::writeXml( QDomElement &element, const QgsReadWriteContext & ) const
 {
   QString str;
+
+  str = QString( "%1" ).arg( mDefaultViewportSize );
```

Comment:
a more straightforward way to convert a number to a string is to use `QString::number( mDefaultViewportSize )`

Reply:
sure! Just need to know these helpers. Thanks